install_config/topics/configuring_rhv_objects.adoc

```diff
@@ -1,66 +1,422 @@
 +
-[cols="1,7"]
-|===
-
-|Master nodes
-a|* Minimum 2 CPU Cores
-* 16 GB Memory
-* Minimum 10 GB root disk
-* Minimum 15 GB container image registry disk
-* 30 GB local volume disk
-* Minimum 25 GB etcd disk
-
-|Infrastructure nodes
-a|* Minimum 2 CPU Cores
-* 16 GB Memory
-* Minimum 10 GB root disk
-* Minimum 15 GB container image registry disk
-* 30 GB local volume disk
-* Minimum 25 GB Gluster registry disk
-
-|Application nodes
-a|* 2 CPU Cores
-* 8 GB Memory
-* Minimum 10 GB root disk
-* Minimum 15 GB container image registry disk
-* 30 GB local volume disk
-
-|Load balancer node
-a|* 1 CPU Core
-* 4 GB Memory
-* 10 GB root disk
-
-|===
-
-. Create master, infrastructure, and application nodes as well as a load
-balancer node. Use the templates that you created.
-
-. Create DNS entries for the routers. Provide entries for all infrastructure
-instances and configure a round-robin strategy so that the router can pass
-traffic to applications.
-
-. Create a DNS entry for the {product-title} web console. Specify the 
-IP address of the load balancer node.
-
-. To use Red Hat Virtualization, you must provide external storage, for example
-GlusterFS, for persistent storage of registry images and for application storage.
+To copy the `<qcow_url>`:
++
+.. Navigate to link:https://access.redhat.com/downloads/content/69/ver=/rhel---7/latest/x86_64/product-software[].
+.. In the *Product Software* tab, locate the *_Red Hat Enterprise Linux KVM Guest Image_*.
+.. Right-click the *Download Now* button and copy the URL.
++
+This link expires after some time. Save the link when you are ready to run the playbook.
+
+. Create the bastion virtual machine:
++
+[options="nowrap" subs="+quotes,verbatim"]
+----
+# ansible-playbook -i localhost create-bastion-machine-playbook.yaml -e @secure_vars.yaml --ask-vault-pass
+----
+
+. Log in to the Administration Portal and click menu:Compute[Virtual Machines] to verify that the *_rhel-bastion_* virtual machine was created successfully.
+
+== Installing {product-title} on Red Hat Virtualization
+
+You can install {product-title} using the bastion virtual machine:
+
+. Log in to *_rhel-bastion_*.
+
+. Create *_vars.yaml_* with the following content and update its parameters:
++
+[source,yml]
+----
+---
+# For detailed documentation of variables, see
+# openshift_ovirt: https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_ovirt#role-variables
+# openshift installation: https://github.com/openshift/openshift-ansible/tree/master/inventory
+engine_url: https://<Manager_VQDN>/ovirt-engine/api
+engine_user: admin@internal
+engine_password: "{{ engine_password }}"
+engine_insecure: false
+engine_cafile: /etc/pki/ovirt-engine/ca.pem
+
+openshift_ovirt_vm_manifest:
+  - name: 'master'
+    count: 1
+    profile: 'master_vm'
+  - name: 'compute'
+    count: 0
```

Comment:
I believe we are missing entry for infra nodes here. For example like this:
```
  - name: infra
    count: 1
    profile: node_vm
```
The fact that it's missing has no effect on the **functionality**, but I think it is still important to provide it here. Because if customer would try to deploy multi-node setup without at least one infra node, it would fail.
@rgolangh @apinnick

Reply:
Done.